exhibition/settings.py

```diff
@@ -20,64 +20,64 @@
 # See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'django-insecure-a_!iokn(zibr0i*@pek7cqs)ou__s-d_h%_-gp)mjj7=so#2(='
+SECRET_KEY = "django-insecure-a_!iokn(zibr0i*@pek7cqs)ou__s-d_h%_-gp)mjj7=so#2(="
```

Comment:
This secret key should be not pushed to repo in real project. We will change it later but just FYI